samples/cpp/tutorial_code/Histograms_Matching/MatchTemplate_Demo.cpp

```diff
@@ -27,11 +25,23 @@ void MatchingMethod( int, void* );
 /**
  * @function main
  */
-int main( int, char** argv )
+int main( int argc, char** argv )
 {
+  if (argc < 3)
+  {
+    cout << "Not enough parameters" << endl;
```

Comment:
If you do this, please provide the correct list of parameters that should be given instead.